accelerator/hpu_accelerator.py

```diff
@@ -186,33 +186,33 @@ def replay_graph(self, graph):
     # Tensor operations
     @property
     def BFloat16Tensor(self):
-        return torch.hpu.BFloat16Tensor
+        return self.hpu.BFloat16Tensor
 
     @property
     def ByteTensor(self):
-        return torch.hpu.ByteTensor
+        return self.hpu.ByteTensor
 
     @property
     def DoubleTensor(self):
-        return torch.hpu.DoubleTensor
+        return self.hpu.DoubleTensor
 
     @property
     def FloatTensor(self):
-        return torch.hpu.FloatTensor
+        return self.hpu.FloatTensor
 
     @property
     def HalfTensor(self):
-        return torch.hpu.HalfTensor
+        return self.hpu.HalfTensor
 
     @property
     def IntTensor(self):
-        return torch.hpu.IntTensor
+        return self.hpu.IntTensor
 
     @property
     def LongTensor(self):
-        return torch.hpu.LongTensor
+        return self.hpu.LongTensor
 
-    def pin_memory(self, tensor, align_bytes=1):
```

Comment:
Why drop `align_bytes`? It is part of the abstraction and used by some accelerators.

Reply:
My mistake. Added it again.